src/Rector/Contrib/PHPUnit/DelegateExceptionArgumentsRector.php

```diff
@@ -29,6 +29,16 @@
         'setExpectedExceptionRegExp' => 'expectExceptionMessageRegExp',
     ];
 
+    /**
+     * @var string[]
+     */
+    private $phpUnitTestCaseClasses = [
+        // PHPUnit 6 and lower
+        'PHPUnit_Framework_TestCase',
+        // PHPUnit 5 and higher
```

Comment:
"PHPUnit 6 and higher"

Reply:
I think it had future compatibility class already in PHPUnit 5.

Reply:
Specific 4.8 and 5.4 versions yes, but this is completed related to PHPUnit 6

Reply:
I see, in that case 5.4+ would make sense.  PHPUnit 4 is dead for long time